requirements_mappings.txt

```
@@ -0,0 +1,3 @@
+# Requirements for property mappings
+tripper>=0.2.5,<0.3
```

```suggestion
tripper>=0.2.10,<0.3
```
Mappings functionality has been moved to `tripper` from v.0.2.10.